components/contract-components/shared/released-by.tsx

```diff
@@ -0,0 +1,38 @@
+import { Flex } from "@chakra-ui/react";
+import {
+  useEnsName,
+  useReleasesFromDeploy,
+} from "components/contract-components/hooks";
+import { useMemo } from "react";
+import { LinkButton, Text } from "tw-components";
+import { shortenIfAddress } from "utils/usedapp-external";
+
+interface ReleasedByProps {
+  contractAddress: string;
+}
+
+export const ReleasedBy: React.FC<ReleasedByProps> = ({ contractAddress }) => {
+  const releasesFromDeploy = useReleasesFromDeploy(contractAddress);
+
+  const lastRelease = useMemo(
+    () => releasesFromDeploy?.data?.at(-1),
```

Comment:
this might not work during SSR (depending on node >16.6), so would prefer to do this with length - 1